tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -17,13 +17,16 @@
 import itertools
 import math
 import warnings
+from collections import namedtuple
 import numpy as np
 
 from ..adapters import Adapter
-from ..config import ConfigValidator, NumberField, StringField, ConfigError
+from ..config import ConfigValidator, NumberField, StringField, ListField, ConfigError
 from ..postprocessor.nms import NMS
 from ..representation import DetectionPrediction
 
+FaceDetectionLayerOutput = namedtuple('FaceDetectionLayerOutput', 'prob_name reg_name anchor_index '+
+                                      'anchor_size win_scale win_length win_trans_x win_trans_y')
```

Comment:
please move closer to adapter. On my view it is more readable if you use list instead of string for fields
```suggestion
FaceDetectionLayerOutput = namedtuple('FaceDetectionLayerOutput',[
                                                                                                                  'prob_name', 
                                                                                                                  'reg_name',
                                                                                                                  'anchor_index',
                                                                                                                   'anchor_size',
                                                                                                                   'win_scale',
                                                                                                                    'win_length',
                                                                                                                    'win_trans_x',
                                                                                                                    'win_trans_y''])
```

Reply:
I agree. Made the changes.